src/messaging/methods/send-email-template.js

```diff
@@ -0,0 +1,36 @@
+import Utils from '../../utils'
+import Urls from '../../urls'
+import Request from '../../request'
+import EmailEnvelope from '../helpers/email-envelope'
+
+export default function sendEmailTemplate(templateName, templateValues, envelopeObject/**, async */) {
+  const responder = Utils.extractResponder(arguments)
+  const isAsync = !!responder
+
+  if (typeof templateName !== 'string') {
+    throw new Error('Template name must be a string')
+  }
+
+  if (templateValues instanceof EmailEnvelope) {
+    envelopeObject = templateValues
+  }
+
+  const data = envelopeObject.toJSON()
```

Comment:
looks like `envelopeObject` can be undefined, can't it?

Reply:
No, it can't be undefined. This method will not be invoked if there are no envelopeObject

Reply:
ok